VMBackup/main/blobwriter.py

```diff
@@ -107,7 +107,9 @@ def WritePageBlob(self, message, blobUri, blobProperties):
                     http_util = HttpUtil(self.hutil)
                     sasuri_obj = urlparse.urlparse(blobUri + '&comp=page')
                     # Get Blob-properties to know content-length
-                    blobContentLength = int(blobProperties.contentLength)
+                    blobContentLength = 0
+                    if(blobProperties is not None):
+                        blobContentLength = int(blobProperties.contentLength)
                     self.hutil.log("WritePageBlob: contentLength:"+str(blobContentLength))
```

Comment:
Is there an different handling that is required for blobProperties being NULL instead of letting the flow go through?

Reply:
skipping the blobWrite if the blob properties are null. this is the current behavior. the change I made will achieve the same without logging an exception.